### PR TITLE
Clarify 2FA usage with desktop and mobile apps

### PR DESCRIPTION
**Pull request by michaelstingl**
We get questions around this a lot, and I usually link to the blog post:
https://owncloud.com/de/news/how-to-use-two-factor-authentication-with-the-owncloud-desktop-client-or-mobile-apps/

But this information should also be available in the docs.

@mmattel @EParzefall please polish

### CODE REVIEW

File: modules/admin_manual/pages/configuration/user/user_auth_twofactor.adoc [hunk 1]
```diff
@@ -16,6 +16,25 @@ If a two-factor provider app is enabled, it is enabled for all users by default
 
 TIP: For the opt-in function to work, ImageMagick and php-imagick are required to create the QR code. Depending on your operating system, these packages may need to be installed manually.
 
+== Using 2FA with ownCloud Desktop and Mobile Apps
+
+ownCloud Desktop and Mobile Apps support 2FA login with browser based login flows.
```
Comment (EParzefall) [inline modules/admin_manual/pages/configuration/user/user_auth_twofactor.adoc:21]:
```suggestion
ownCloud Desktop and Mobile Apps support 2FA login with browser-based login flows.
```

File: modules/admin_manual/pages/configuration/user/user_auth_twofactor.adoc [hunk 2]
```diff
@@ -16,6 +16,25 @@ If a two-factor provider app is enabled, it is enabled for all users by default
...
+
+ownCloud Desktop client will open the login page in the system web browser, and ownCloud Mobile Apps will open the login page in an embedded web view. After entering the regular credentials, users will see a second page, where they need to enter the second factor.
+
+Here you can find more information:
+
+- https://doc.owncloud.com/server/next/admin_manual/configuration/user/user_oauth2.html
+
+=== App Passwords / Tokens
+
+Without OAuth 2.0 App, users need to login to their ownCloud account in regular web browser first, then create an app password / tokens, which then can be used in the ownCloud Desktop and Mobile apps.
```
Comment (EParzefall) [inline modules/admin_manual/pages/configuration/user/user_auth_twofactor.adoc:33]:
```suggestion
Without the OAuth 2.0 App, users need to log in to their ownCloud account in a regular web browser first, then create an app password or tokens, which can be used in the ownCloud Desktop and Mobile apps.
```

**Review by EParzefall — APPROVED** [review]

**Comment by EParzefall** [comment]
@michaelstingl  Backports to 10.11 and 10.10?

**Comment by michaelstingl** [comment]
> @michaelstingl Backports to 10.11 and 10.10?

Yes, nothing new.

**Pull request merged**